packages/pubsub/src/subscription.js

```diff
@@ -87,6 +77,21 @@ var PUBSUB_API_TIMEOUT = 90000;
  * @alias module:pubsub/subscription
  * @constructor
  *
+ * @param {object=} options - See a
+ *     [Subscription resource](https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.subscriptions)
+ * @param {number} options.ackDeadline - The maximum time after receiving a
+ *     message that you must ack a message before it is redelivered.
```

Comment:
It looks to me like the automatic inventory management plows over this (meaning maybe this option should disappear).